### PR TITLE
[zh-cn] sync blog 2023-04-20-read-write-once-pod-access-mode-beta.md

### PR DESCRIPTION
**Pull request by asa3311**
sync blog 2023-04-20-read-write-once-pod-access-mode-beta.md
link #40710

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+[What are access modes and why are they important?](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/#what-are-access-modes-and-why-are-they-important)
+in the _Introducing Single Pod Access Mode for PersistentVolumes_ article from 2021.
+-->
+要了解有关访问模式和 ReadWriteOncePod 如何工作的更多背景信息，
+请阅读2021年介绍持久卷的单个Pod访问模式的文章中的[什么是访问模式和为什么它们如此重要？](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/#what-are-access-modes-and-why-are-they-important)。
+
+<!--
+## Changes in the ReadWriteOncePod beta
+-->
+## ReadWriteOncePod Beta版的变化  {#changes-in-the-readwriteoncepod-beta}
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:60]:
```suggestion
## ReadWriteOncePod 的 Beta 版中变化  {#changes-in-the-readwriteoncepod-beta}
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+<!--
+The ReadWriteOncePod beta adds support for
+[scheduler preemption](/docs/concepts/scheduling-eviction/pod-priority-preemption/)
+of pods using ReadWriteOncePod PVCs.
+-->
+ReadWriteOncePod Beta 版支持 ReadWriteOncePod PVCs 的 Pod 添加[调度器抢占](/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption/)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:67]:
```suggestion
ReadWriteOncePod Beta 版为使用 ReadWriteOncePod PVC 的 Pod
添加[调度器抢占](/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption/)支持。
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+ReadWriteOncePod PVCs can also be preempted if a higher-priority pod requires
+the same PVC.
+-->
+调度器抢占允许更高优先级的 Pod 抢占较低优先级的 Pod，以便它们可以在同一节点上运行。
+在此版本中，如果更高优先级的 Pod 需要相同的 PVC，使用 ReadWriteOncePod PVCs 的 Pod 也可以被抢占。
+
+<!--
+## How can I start using ReadWriteOncePod?
+-->
+## 如何开始使用 ReadWriteOncePod ？  {#how-can-i-start-using-readwriteoncepod}
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:81]:
```suggestion
## 如何开始使用 ReadWriteOncePod？  {#how-can-i-start-using-readwriteoncepod}
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+随着 ReadWriteOncePod 现已升级为 Beta 版，在 v1.27 及更高版本的集群中将默认启用该功能。
+
+<!--
+Note that ReadWriteOncePod is
+[only supported for CSI volumes](/docs/concepts/storage/persistent-volumes/#access-modes).
+Before using this feature you will need to update the following
+[CSI sidecars](https://kubernetes-csi.github.io/docs/sidecar-containers.html)
+to these versions or greater:
+-->
+请注意，ReadWriteOncePod [仅支持CSI卷](/zh-cn/docs/concepts/storage/persistent-volumes/#access-modes)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:96]:
```suggestion
请注意，ReadWriteOncePod [仅支持 CSI 卷](/zh-cn/docs/concepts/storage/persistent-volumes/#access-modes)。
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+在使用此功能之前，您需要将以下 [CSI sidecars](https://kubernetes-csi.github.io/docs/sidecar-containers.html)更新至以下版本或更高版本：
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:97]:
```suggestion
在使用此功能之前，你需要将以下 [CSI Sidecars](https://kubernetes-csi.github.io/docs/sidecar-containers.html)更新至以下版本或更高版本：
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+要开始使用 ReadWriteOncePod，请创建具有 ReadWriteOncePod 访问模式的 PVC：
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: single-writer-only
+spec:
+  accessModes:
+  - ReadWriteOncePod # Allow only a single pod to access single-writer-only.
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:120]: 翻译注释

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+
+<!--
+If your storage plugin supports
+[dynamic provisioning](/docs/concepts/storage/dynamic-provisioning/),
+new PersistentVolumes will be created with the ReadWriteOncePod access mode applied.
+
+Read [Migrating existing PersistentVolumes](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/#migrating-existing-persistentvolumes)
+for details on migrating existing volumes to use ReadWriteOncePod.
+-->
+如果你的存储插件支持[动态供应](/zh-cn/docs/concepts/storage/dynamic-provisioning/)，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:134]:
```suggestion
如果你的存储插件支持[动态制备](/zh-cn/docs/concepts/storage/dynamic-provisioning/)，
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+将会创建具有 ReadWriteOncePod 访问模式的新持久卷。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:135]: 缺少主语

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+
+阅读[迁移现有持久卷](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/#migrating-existing-persistentvolumes)以了解如何将现有卷迁移和使用 ReadWriteOncePod。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:137]:
```suggestion
阅读[迁移现有持久卷](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/#migrating-existing-persistentvolumes)
以了解如何迁移现有卷以使用 ReadWriteOncePod。
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+## How can I learn more?
+-->
+## 如何了解更多信息？ {#how-can-i-learn-more}
+
+<!--
+Please see the [alpha blog post](/blog/2021/09/13/read-write-once-pod-access-mode-alpha)
+and [KEP-2485](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md)
+for more details on the ReadWriteOncePod access mode and motivations for CSI spec changes.
+-->
+请查看 [alpha 版博客](/blog/2021/09/13/read-write-once-pod-access-mode-alpha)和
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:149]:
```suggestion
请查看 [Alpha 版博客](/blog/2021/09/13/read-write-once-pod-access-mode-alpha)和
```

File: content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+[KEP-2485](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md)
+以了解关于 ReadWriteOncePod 访问模式的更多详细信息以及对 CSI 规范更改的动机。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-04-20-read-write-once-pod-access-mode-beta.md:151]:
```suggestion
以了解关于 ReadWriteOncePod 访问模式的更多详细信息以及对 CSI 规约作更改的动机。
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**